test/accel/IntegrationTestBase.cc

```diff
@@ -386,13 +399,23 @@ auto LarSphereIntegrationMixin::make_sens_det(std::string const& sd_name)
  */
 void LarSphereIntegrationMixin::process_hit(G4Step const* step)
 {
-    ASSERT_TRUE(step);
-    ASSERT_TRUE(step->GetTrack());
+    if (!step || !step->GetTrack())
```

Comment:
This seems off. If `step = nullptr` we assert that it is not a `nullptr`?
We either test that every `step*` is in a valid state, or if we are sure they will be, we can just proceed with the `EXPECT` calls, no?